eng/pipelines/runtime.yml

```diff
@@ -293,6 +297,7 @@ jobs:
       condition: >-
         or(
           eq(dependencies.checkout.outputs['SetPathVars_libraries.containsChange'], true),
+          eq(dependencies.checkout.outputs['SetPathVars_runtimetests.containsChange'], true),
```

Comment:
I guess building mono doesn't take long, but it still uses machines and cores and with all the core reduction we're trying to do, should we separate the OSX build into it's own entry? It seems overkill to build all of mono when runtime tests changes other than building only the mono matrix we need to run the runtime tests.

Reply:
I have another change I am working on to run these tests on more architectures. But maybe not all of them; I will make the change.